account_invoice_start_end_dates/models/account_invoice.py

```diff
@@ -0,0 +1,92 @@
+# Copyright 2013-2016 Akretion, Alexis de Lattre <alexis.delattre@akretion.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import models, fields, api, _
```

Comment:
i am pretty happy with using isort for sorting the imports. May be you can try it : ))) https://pypi.org/project/isort/
It orders the imports in the right way also compatible with the contribution guidelines [1]

[1] https://github.com/OCA/odoo-community.org/blob/master/website/Contribution/CONTRIBUTING.rst

Reply:
it does seem to move the _ to the front, but I guess that's alphabetically correct, thanks for the tip!